introduction_to_applying_machine_learning/ntm_20newsgroups_topic_modeling/ntm_20newsgroups_topic_model.ipynb

```diff
@@ -1018,16 +1092,18 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "collapsed": true
+   },
    "outputs": [],
    "source": []
   }
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "conda_mxnet_p36",
+   "display_name": "Python 3",
    "language": "python",
-   "name": "conda_mxnet_p36"
+   "name": "python3"
```

Comment:
I think in SageMaker notebook there's not a kernel called 'Python 3' or 'python3'. The name should be 'conda_python3'. You could confirm that in other example notebooks.

Reply:
will correct